test/utils/utils.ts

```diff
@@ -19,6 +19,14 @@ export function assertContains(output: string, pattern: string) {
     }
 }
 
+export function assertEquals(actual: string, expected: string) {
```

Comment:
Why not use the `assertEqual` function already defined in this file?

Reply:
Aah, shultz...
That was mighty retarded... 🤦

Reply:
Fixed